lib/config.js

```diff
@@ -25,6 +26,19 @@ const ModeIcon = ClassicModeIcon
  */
 const ItineraryFooter = () => <div />
 
+/**
+ * Renders flag icons for supported languages
+ * using emojis would be a simpler solution, but they are not available on Windows
+ */
+export const FLAG_ICON_MAPPING = {
+  'en-US': <US style={{ width: 15 }} />,
+  'es-ES': <ES style={{ width: 15 }} />,
+  'fr-FR': <FR style={{ width: 15 }} />,
+  'ko-KR': <KR style={{ width: 15 }} />,
+  'vi-VN': <VN style={{ width: 15 }} />,
+  'zh-CN': <CN style={{ width: 15 }} />
+}
```

Comment:
As mentioned earlier, these defaults need to go somewhere else

Reply:
Updated to be a constant in the locale-selector file in https://github.com/opentripplanner/otp-react-redux/pull/525/commits/a6de95114aed8b19ec7c8910f2e5e500a88a524d